build/localnet/manifests/configs.yaml

```diff
@@ -28,11 +28,11 @@ data:
         "block_store_path": "/validator-storage/blockstore",
         "tx_indexer_path": "",
         "trees_store_dir": "/validator-storage/trees",
-        "max_conns_count": 8,
-        "min_conns_count": 0,
-        "max_conn_lifetime": "1h",
-        "max_conn_idle_time": "30m",
-        "health_check_period": "5m"
+        "max_conns_count": 50,
+        "min_conns_count": 1,
+        "max_conn_lifetime": "5h",
+        "max_conn_idle_time": "2m",
+        "health_check_period": "30s"
```

Comment:
There's inconsistency between these values and the ones in `build/config/config{1..4}.json` is that on purpose?